recipes/uv-secure/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "uv-secure" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/uv_secure-{{ version }}.tar.gz
+  sha256: 0aeeb2e27ebf9f5d1fef7050a1948055308cc48a0567a89972f2212b9dfcc2ba
+
+build:
+  skip: true  # [py<39]
+  entry_points:
+    - uv-secure = uv_secure.run:app
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - hatchling
+    - pip
+  run:
+    - python
+    - anyio >=4.7.0
+    - asyncer >=0.0.8
+    - hishel >=0.1.1
+    - httpx >=0.28.1
+    - humanize >=4.11.0
+    - inflect >=7.4.0
+    - pydantic >=2.10.3
+    - rich >=13.9.4
+    - tomli  # [py<311]
```

You can add a comment to drop this when updating the min Python to 3.11, but for now it is better to install tomli in later Pythons so we can have this as noarch.

```suggestion
    - tomli
```